src/sbt-test/sbt-scalafix/scalafixResolvers/test

```diff
@@ -0,0 +1,3 @@
+# reload to ensure env variable COURSIER_REPOSITORIES is picked up by scalafixResolvers
+> reload
```

Comment:
out of curiosity, is the reload required by `sbt-dotenv` for the env file to be applied?